packages/block-editor/src/components/inserter/block-patterns-filter.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	SVG,
+	Path,
+	DropdownMenu,
+	MenuGroup,
+	MenuItemsChoice,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/icons';
+import { useState } from '@wordpress/element';
+
+export const PATTERN_TYPES = {
```

Comment:
The plan when we added the new constants to the patterns package was to then tidy up the block editor to use the same constants, but it is the editor that currently depends on the patterns package rather than the block editor so not sure if this will be possible - it would be good to try and work out how all the pattern related constants can be centralised somehow.

Reply:
> That might not be trivial as the block-editor might not be able to depend on those packages.

Yeah, that's the reason I didn't touch these either. 😄 

To be honest, my only motivation was to clean up the site editor package. There were some constants being used, others duplicated etc. I think it's okay to have some dupes across packages, but the site editor needed some spring cleaning IMO